docs/versioned-plugins/include/6.x/version-list-intro.asciidoc

```diff
@@ -1,6 +1,7 @@
 [id="{type}-{plugin}-index"]
```

Comment:
ditto. this is copied from the logstash repo. Make sure you also apply the changes to the file in the logstash repo, too.